internal/agent/install.go

```diff
@@ -298,16 +282,56 @@ func RunInstall(options map[string]string) error {
 		fmt.Printf("could not write cloud init: %s\n", err.Error())
 		os.Exit(1)
 	}
-	args := []string{"install"}
-	args = append(args, optsToArgs(options)...)
-	args = append(args, "-c", f.Name(), device)
 
-	cmd := exec.Command("elemental", args...)
-	cmd.Env = os.Environ()
-	cmd.Stdout = os.Stdout
-	cmd.Stdin = os.Stdin
-	cmd.Stderr = os.Stderr
-	if err := cmd.Run(); err != nil {
+	// Load the installation Config from the system
+	// TODO: This uses the default mounter, logger, fs, etc...
+	// Make it configurable?
+	installConfig, err := elementalConfig.ReadConfigRun("/etc/elemental")
+	if err != nil {
+		return err
+	}
+
+	// Set our cloud-init to the file we just created
+
+	_, reboot := options["reboot"]
+	_, poweroff := options["poweroff"]
```

Comment:
```suggestion
	reboot := options["reboot"]
	poweroff := options["poweroff"]
```

Reply:
this is a map[string]string], so cant get just that without more work afterwards. Doing it trhis way means that the reboot and poweroff are directly a bool type :)

Reply:
could `options["reboot"]` be set to `"false"` ? Because in that case `reboot` would be `true` (because the key exists), although it's value is "false"

Reply:
That is a good point!

Reply:
welp, this actually does the same behaviour that the old one! so yeah, not touching this in the PR, maybe afterwards as its a behaviour change!